chroniker/models.py

```diff
@@ -298,28 +298,28 @@ def due_with_met_dependencies(self, jobs=None):
             valid = True
 
             if job.check_is_running():
-                #print('Skipping job %i (%s) which is already running.' % (job.id, job))
+                # print('Skipping job %i (%s) which is already running.' % (job.id, job))
```

Comment:
Note, you don't need to waste time reformatting dead code like this. It just adds clutter to the PR. These should just be removed.